src/views/SelectJobsModal.vue

```diff
@@ -12,7 +12,11 @@
   <ion-content>
     <ion-searchbar v-model="queryString" :placeholder="$t('Search jobs')" @keyup.enter="search($event)" />
 
-    <div v-if="jobs.length === 0" class="ion-text-center">
+    <div v-if="!isJobSearched" class="ion-text-center">
+      <p>{{ $t("Searched jobs will appear here") }}</p>
+    </div>    
+
+    <div v-else-if="jobs.length === 0" class="ion-text-center">
```

Comment:
Use `queryString` instead of `isJobSearched`

Reply:
As queryString is v-modelled with the search bar, even if we type something, without searching (pressing enter) it'll show 'No jobs found' in that case.

Reply:
Instead of using v-model, we could update the queryString only on enter press. This way we will not need flag variable